src/GitList/Git/Client.php

```diff
@@ -53,8 +53,14 @@ public function getRepositories($paths)
         return $allRepositories;
     }
 
-    private function recurseDirectory($path, $topLevel = true)
+    private function recurseDirectory($path, $topLevel = true, $basePath = "")
     {
+        if ($topLevel) {
+            $basePath = realpath($path) . DIRECTORY_SEPARATOR;
```

Comment:
I had to add additional code because it caused the template to generate url like /repository_root/repo_path.
```
+  $repoName = preg_replace('~^'. preg_quote($basePath, '~') .'~', '', $repoName);

   $repositories[$repoName] = array(
                           'name' => $repoName,
                           'path' => $file->getPathname(),
                           'description' => $description
                       );
```